src/utils.ts

```diff
@@ -142,3 +144,31 @@ export class JobIdFilterDescriptor extends FilterDescriptor {
         "Job Id search"));
     }
 }
+
+/*************************************************************************************************************
+ * Returns array of all subnodes of given node
+ *************************************************************************************************************/
+export async function concatUSSChildNodes(nodes: ZoweUSSNode[]) {
+    let allNodes = new Array<ZoweUSSNode>();
+
+    for (const node of nodes) {
+        allNodes = allNodes.concat(await concatUSSChildNodes(await node.getChildren()));
+        allNodes.push(node);
+    }
+
+    return allNodes;
+}
```

Comment:
Why not "extend" the `concatChildNodes` function instead of duplicating thins?

Reply:
I thought we went to node.children instead of getChildren?

Reply:
@Colin-Stone I switched to `node.children` 😄 
@zFernand0 Seems that now test coverage delta is 0 😄

Reply:
@zFernand0 Re: Why not "extend" the concatChildNodes function instead of duplicating thins? 
I hope to abstract the node class in the next PI so this could be a single method in there.

Reply:
Unit test Coverage is an issue in extension.ts. Currently master is showing  extension.ts         
`|    83.21 |    81.33 |    35.35 |    91.55 |... 1738,1783,1795 |`
your new branch has dropped this to  extension.ts         
`|    82.04 |    79.27 |    35.71 |    89.62 |... 1801,1842,1856 |`